api/proxy.config.ts

```diff
@@ -116,6 +117,13 @@ export const initProxy = (app: Express) => {
         target: getConfigValue(SERVICES_PAYMENTS_URL),
     })
 
+    applyProxy(app, {
+        rewrite: true,
+        rewriteUrl: "/refund",
```

Comment:
@ronaldmansveld - I'm being really pedantic here but can we have single quotes instead of double (given they're single everywhere else)?